settings/config.settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+// Ensure the appropriate config split is enabled.
+$config['config_split.config_split.local']['status'] = FALSE;
+$config['config_split.config_split.dev']['status'] = FALSE;
+$config['config_split.config_split.stage']['status'] = FALSE;
+$config['config_split.config_split.prod']['status'] = FALSE;
+$config['config_split.config_split.ci']['status'] = FALSE;
```

Comment:
@grasmash - There are no extra settings for RA (many subscriptions have an `ra` environment as well). Do you think that might be something to add? Or just note that people should add an ra-specific split in their own settings.php if they need it?

One other note—for my sites I'm currently using a `test` split (not `stage`) to match the fact that Acquia Cloud still uses `test` as the machine name for the `Stage` environment... for those projects who are already set up using this kind of naming convention, I'll try to document how to switch the split name correctly. Just wanted to put that out there.